### PR TITLE
SKO overview page Cleanup

### PR DESCRIPTION
**Pull request by Timo-Breumelhof**
- Removed SKO that don't work any more
- Added SKOs that were not listed
- Moved legacy SKOs to separate table
- Moved SKOs that are better done using modern techniques to separate table

### CODE REVIEW

**Comment by Timo-Breumelhof** [comment]
Sorry I needed 9 commits for this :-)

File: content/tutorials/themes/theme-objects/index.md [hunk 1]
```diff
@@ -113,13 +113,11 @@ Where the "Register" block should be on top of the ASCX file.
 The <dnn:BREADCRUMB...> block should be placed in the Theme on the spot you want it to appear.
 
 > [!NOTE]
-> The following list is far from complete (some work to be done) and some of the Theme Objects in the list are legacy
+> The following list are the Core Theme / Container Objects.
 
 |**Theme Object**|**Description**|
 |---|---|
-|ACTIONBUTTON|Displays an action from the **module action menu**.|
 |BREADCRUMB|Displays the path to the current tab (`>` is the default separator). Example: `PageName1 > PageName2 > PageName3`|
```
Comment (david-poindexter) [inline content/tutorials/themes/theme-objects/index.md]: Here is where you would link to the new page from #473

Comment (david-poindexter) [inline content/tutorials/themes/theme-objects/index.md]: Here is where you would link `BREADCRUMB` to the new page in #473

Comment (Timo-Breumelhof) [reply content/tutorials/themes/theme-objects/index.md]: @david-poindexter done at #473

File: content/tutorials/themes/theme-objects/index.md [hunk 2]
```diff
@@ -128,27 +129,40 @@ The <dnn:BREADCRUMB...> block should be placed in the Theme on the spot you want
...
+|JAVASCRIPTLIBRARYINCLUDE|Include an installed Javascripot Library|
 |LANGUAGE|Displays the language selector dropdown list or the language flags based on the theme object attribute settings.|
-|LEFTMENU|Displays a vertical menu layout.|
-|LINKS|Displays a flat menu of links associated with the current tab level and the parent node.|
+|LINKTOFULLSITE|Displays a link to the Desktop version of the website|
+|LINKTOMOBILE|Displays a link to the Mobile version of the website|
 |LOGIN|Displays **Login** for anonymous users and **Logout** for authenticated users.|
 |LOGO|Displays the website's logo.|
-|NAV|Displays a menu according to the type specified in the ProviderName attribute.|
-|PRINTMODULE|Displays a link for the Print action from the **module action menu**.|
```
Comment (david-poindexter) [inline content/tutorials/themes/theme-objects/index.md]: @Timo-Breumelhof AFAIK `PRINTMODULE` works again now.

Comment (Timo-Breumelhof) [reply content/tutorials/themes/theme-objects/index.md:142]: @david-poindexter ok done

**Pull request merged**